src/core/main_menu.cpp

```diff
@@ -63,6 +83,9 @@ void wifiOptions() {
   options.push_back({"Scan Hosts", [=]()    { local_scan_setup(); }});
 #ifndef LITE_VERSION
   options.push_back({"Wireguard", [=]()     { wg_setup(); }});
+#endif
+#ifdef CARDPUTER
```

Comment:
If it is to be compatible with all devices, use the following check:
```
#ifndef LITE_VERSION // it is to be used to limit features for devices to sabe memory.
....
#endif
```

Reply:
I already tried with ifndef LITE_VERSION, but it seems that StickC Plus 2 is not a LITE_VERSION.
And it does not work with StickC Plus 2.

Perhaps it could be good to add Sticks to LITE version.

Reply:
Excluding the M5Unified.h lib doesn't fix it?

Reply:
Unfortunately no =(